muffnn/mlp/mlp_classifier.py

```diff
@@ -262,3 +262,8 @@ def __getstate__(self):
             state['n_classes_'] = self.n_classes_
 
         return state
+
+    def score(self, X, y):
+        accuracy = np.array(y) == self.predict(X)
+        accuracy = accuracy[np.array(y) != -1].mean(axis=0).mean()
```

Comment:
This average is per label. I do not remember. Isn't the loss computed as an average over the non-missing labels for each example and then an average over the examples? We might want to do that here too if so.